03/app.js

```diff
@@ -1,3 +1,14 @@
-const x = 10;
+const x = 5;
 let iteration = 0;
-let randomNumber = -1;
\ No newline at end of file
+let randomNumber = 0;
+
+
+
+while (x !== randomNumber) {
+    randomNumber = Math.round(Math.random() * 10);
+    iteration++
+    if (randomNumber === x) {
+        console.log(`after checking ${iteration} times, We found equal number to x`);
+       break;
+    }
```

Comment:
Czy faktycznie ta część jest potrzebna? Zwróć uwagę, żę warunek już masz w linii 7 dlatego prościć można to zrobić tak:

```
while (x !== randomNumber) {
    randomNumber = Math.round(Math.random() * 10);
    iteration++
}
// jeśli wychodzimy z pętli to to oznacza, że wylosowaliśmy x :)
console.log(`after checking ${iteration} times, We found equal number to x`);
```

Reply:
Jak najbardziej masz rację, za dużo kombinowania :P